ecs-cli/modules/cli/cluster/userdata/user_data.go

```diff
@@ -143,21 +143,26 @@ func isMultipart(data string) (bool, map[string]string, io.Reader) {
 	return false, nil, nil
 }
 
-func (b *Builder) getClusterUserData() string {
+func (b *Builder) getClusterUserData(enableTagging bool) string {
```

Comment:
instead of passing the `bool` to each private method, is it possible to have `Build()` set it as a property on the `UserDataBuilder` and then referencing that property instead of a param?

Reply:
I like the idea, I'm adding it in the constructor tho instead